app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -383,6 +397,102 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
         }
     }
 
+    //Settings
+    private void showSettingsWidget() {
+        if (mSettingsWidget == null) {
+            mSettingsWidget = new SettingsWidget(this);
+            mSettingsWidget.getPlacement().parentHandle = mBrowserWidget.getHandle();
+            mSettingsWidget.setDelegate(new SettingsWidget.Delegate() {
+                @Override
+                public void onSettingsCrashReportingChange(boolean isEnabled) {
+                    SettingsStore.getInstance(VRBrowserActivity.this).setCrashReportingEnabled(isEnabled);
+                    SessionStore.get().getSettings().setRemoteDebuggingEnabled(isEnabled)
```

Comment:
Should we enable remote debugging when crash reporting is enabled?

Reply:
Remote debugging will need to be a separate setting and is not related to crash reporting.